### ISSUE TITLE
Bedfile support?

### ISSUE THREAD

**Issue by williakd17** [alizrrao/DepthOfCoveragePerInterval#1]
Hello,

Does the depth_of_coverage_per_interval.py accept .bed files? 

Thank you

**Comment by alizraksi** [comment]
No, it doesn't, but you can easily convert a BED file to an interval file using the Picard tool 
BedToIntervalList, details here:
https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.0.0/picard_util_BedToIntervalList.php

Anyway that's a good idea to add BED file support in the future.
